test/e2e/lib/gutenberg/tracking/general-tests.js

```diff
@@ -158,6 +158,32 @@ export function createGeneralTests( { it, editorType, postType } ) {
 		);
 	} );
 
+	it( 'Tracks "wpcom_block_editor_nav_sidebar_open" event', async function () {
+		const editor = await EditorComponent.Expect( this.driver, gutenbergEditorType );
```

Comment:
I expect this test will fail for site editor until https://github.com/Automattic/wp-calypso/pull/53500 also lands?

Reply:
Yep! That's correct, updated PR desc.